set.mm

```diff
@@ -50217,9 +50217,10 @@ That theorem bundles the theorems ( ` |- E. x ( x = y -> z e. x ) ` with
   univ $p |- U. _V = _V $=
     ( cvv cpw cuni pwv unieqi unipw eqtr3i ) ABZCACAHADEAFG $.
 
-  $( Membership of a power class.  Exercise 10 of [Enderton] p. 26.
-     (Contributed by NM, 13-Jan-2007.) $)
-  pwel $p |- ( A e. B -> ~P A e. ~P ~P U. B ) $=
+  $( Quantitative version of ~ pwexg : the powerset of an element of a class is
+     an element of the double powerclass of the union of that class.  Exercise
+     10 of [Enderton] p. 26.  (Contributed by NM, 13-Jan-2007.) $)
+  pwel $p |- ( A e. V -> ~P A e. ~P ~P U. V ) $=
```

Comment:
Why did you change ` B ` to ` V `? We usually use ` V ` for an arbitrary class, especially in the idioms ` A e. V ` to express "`A ` is a set". In this case, however, ` V `is not arbitrary, because it appears at the RHS of the implication ("that class").  Therefore, I thinlk ` B ` is a better choice here.

Reply:
Done.